packages/server/api/src/app/ai/mcp/docs-tools.ts

```diff
@@ -28,7 +28,7 @@ export async function getDocsTools(): Promise<MCPTool> {
   const searchTool = tools['search'];
 
   const toolSet = {
-    docsMcpClient: tool({
+    OpenOps_Docs: tool({
```

Comment:
If that's what's going to be displayed in the UI, then please use OpenOps_Documentation